### ISSUE TITLE
Accessibility violation triggered by  "Search Form or Search Results" menu item: Form elements must have labels

### ISSUE THREAD

**Issue by eddiekonczal** [joomla/joomla-cms#25127]
### Steps to reproduce the issue

Create a "Search Form or Search Results" menu item

Audit the web page with an accessibility tool such as Deque WorldSpace plugin for Chrome

### Expected result

The accessibility tool should not find any violations.

### Actual result

The following violation is reported:

Form elements must have labels

Issue description
Ensures every form element has a label

Impact: critical
 Learn more
Element location
input[autocomplete="off"]
Element source
<input type="text" autocomplete="off" readonly="">
To solve this violation, you need to:
Fix at least one (1) of these issues:
aria-label attribute does not exist or is empty

aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty

Form element does not have an implicit (wrapped) <label>

Form element does not have an explicit <label>

Element has no title attribute or the title attribute is empty

### System information (as much as possible)

PHP Built On 	Linux 4.4.162-94.72-default #1 SMP Mon Nov 12 18:57:45 UTC 2018 (9de753f) x86_64
Database Type 	mysql
Database Version 	5.5.39
Database Collation 	utf8_general_ci
Database Connection Collation 	utf8mb4_general_ci
PHP Version 	7.0.7
Web Server 	Apache
WebServer to PHP Interface 	apache2handler
Joomla! Version 	Joomla! 3.9.6 Stable [ Amani ] 7-May-2019 15:00 GMT
Joomla! Platform Version 	Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent 	Mozilla/5.0 (Windows NT 10.0; WOW64; rv:52.0) Gecko/20100101 Firefox/52.0 SeaMonkey/2.49.4

**Comment by brianteeman** [comment]
That is coming from the dropdown for ordering and unfortunately cannot be fixed as it is a problem with the js library used for that dropdown. The only thing I can suggest is to either disable the "Use search options" or to create a template override.

![image](https://user-images.githubusercontent.com/1296369/58972885-819f2900-87b6-11e9-9fac-46e781213c02.png)

**Comment by SharkyKZ** [comment]
If this is about search input, it's fixed in 3.9.7 with https://github.com/joomla/joomla-cms/pull/24482.

**Comment by brianteeman** [comment]
No its the two dropdowns that use chosen 
![image](https://user-images.githubusercontent.com/1296369/59334916-6fac1180-8cf3-11e9-8622-c59e0c0b2838.png)

**Comment by SharkyKZ** [comment]
OK. Based on discussion in #24623 this won't be fixed in 3.x?

**Comment by HLeithner** [comment]
I reverted my decision in https://github.com/joomla/joomla-cms/pull/24627 so if brian can restore his pr https://github.com/joomla/joomla-cms/pull/24623 I will merge it in 3.9.9

**Comment by ghost** [comment]
@HLeithner can you please change Milestone 3.9.8 > 3.9.9?

**Comment by brianteeman** [comment]
@HLeithner All you have to do is to reopen the PR the branch is still there (I dont have access to re-open)

**Comment by HLeithner** [comment]
done thx

**Comment by SharkyKZ** [comment]
This can be closed since https://github.com/joomla/joomla-cms/pull/24623 has been merged.
